src/cli.ls

```diff
@@ -76,7 +76,7 @@ if program.macros
 
 
 if program.body and (program.style or program.url)
-    console.error "error: conflicting options: 'url' and 'style' cannot be used with 'body'!"
+    console.error "error: conflicting options: 'url' and 'stylesheet' cannot be used with 'body'!"
```

Comment:
LOL :grin: this is good, but it still lacks line 78 :)

Reply:
Thanks :laughing: